crates/p2p/tests/gossipsub.rs

```diff
@@ -32,7 +31,7 @@ struct Setup {
 impl Setup {
     /// Spawn N operators that are connected "all-to-all" with handles to them, task tracker
     /// to stop control async tasks they are spawned in.
-    pub async fn all_to_all(number: usize) -> Result<Self, snafu::Whatever> {
+    pub async fn all_to_all(number: usize) -> Result<Self, Box<dyn std::error::Error>> {
```

Comment:
In tests, we can use `anyhow`, with its similar to `whatever!` from `snafu` macros

Reply:
I've added anyhow usages with it's `bail!`, but in certain places occur compilation errors about unsafe shared between threads etc, thus there I've left boxed variant

Reply:
That's strange at least

Reply:
Can you point at place of code, where I'll get this error if I change `Box<dyn Error>` to `anyhow::Error` ?